src/Criteria.php

```diff
@@ -155,16 +155,19 @@ public function getWhereExpression()
      */
```

Comment:
In the future, we would have `@return array<string, Order>` here

Reply:
Maybe I should use `array<string, string|Order>` right now so that people start adapting their code, even if in practice we don't break BC and stay with `array<string, string>` ? :thinking:

Reply:
We should provide a way to opt into the future behavior of this method. We could do so by adding a boolean flag (meh) or by adding a new method and deprecating this one.

Reply:
Ok, I'll try to find a new name.

Reply:
Since there is no setter, I went with just `orderings()`